src/views/GovernanceReviewTeam/index.tsx

```diff
@@ -63,6 +71,70 @@ const GovernanceReviewTeam = () => {
     ? `${systemIntake.requester.name}, ${component.acronym}`
     : systemIntake.requester.name;
 
+  // Get admin lead assigned to intake
+  const getAdminLead = () => {
+    if (systemIntake.adminLead) {
+      return systemIntake.adminLead;
+    }
+    return t('governanceReviewTeam:adminLeads.notAssigned');
+  };
+
+  const [newAdminLead, setAdminLead] = useState('');
```

Comment:
Based on some research I did, I suspect it might be slightly overkill to use the useState hook for this purpose but I could be mistaken. I tried to make it a more simple pattern but couldn't get it to work properly. Happy to change it if there is a better pattern for this.